README.md

```diff
@@ -103,18 +106,18 @@ Setup for developers (Windows)
     $ \c systersdb;
     $ GRANT ALL PRIVILEGES ON systersdb TO <username created above>;
     ```
-1. Fill in the database details in `systers_portal/settings/dev.py`.
-1. Run `set SECRET_KEY=foobarbaz` in your terminal, ideally the secret key
-  should be 40 characters long, unique and unpredictable. 
-1. Run `python systers_portal/manage.py migrate`.
-1. Run `python systers_portal/manage.py cities_light` for downloading and importing data for django-cities-light.
-1. Run `python systers_portal/manage.py createsuperuser` to create a superuser for the admin panel.
+8. Fill in the database details in `systers_portal/systers_portal/settings/dev.py`.Fill `Username` and `Password` values set while creating `systerdb` database in Step 7.
```

Comment:
Very Minor Changes: No space before the full stop, some steps have full stops and some don't. Otherwise looks great!!

Reply:
Updated!